lib/dtutils/file.lua

```diff
@@ -291,9 +291,13 @@ dtutils_file.libdoc.functions["file_copy"] = {
 function dtutils_file.file_copy(fromFile, toFile)
   local result = nil
   -- if cp exists, use it
-  if dtutils_file.check_if_bin_exists("cp") then
+  if dt.configuration.running_os == "windows" then
+    os.execute('copy "' .. fromFile .. '" "' .. toFile .. '"')
+    result = true
```

Comment:
this should be result = os.execute also.  It returns nil on failure and true on success, so we can use the windows copy command and if it fails for some reason then fall back to the pure lua copy solution.

Reply:
done